syz-manager/rpc.go

```diff
@@ -306,3 +308,11 @@ func (serv *RPCServer) Poll(a *rpctype.PollArgs, r *rpctype.PollRes) error {
 		a.Name, len(r.Candidates), len(r.NewInputs), len(r.MaxSignal.Elems))
 	return nil
 }
+
+func (serv *RPCServer) getMachineInfo(name string) ([]byte, bool) {
+	fuzzer, ok := serv.fuzzers[name]
+	if !ok {
+		return nil, false
+	}
+	return fuzzer.machineInfo, true
```

Comment:
As I mentioned, it would be good to reset serv.fuzzers[name] to nil here. So that we don't leave stale machine info from the previous run that we can wrongly fetch next time, if VM crashes before fuzzer has started and sent us new machine info.